CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.0.1.dev6 (2018-06-21)
```

Comment:
dev*7*

Reply:
I'm really not on top of my game today...